src/core/core.controller.js

```diff
@@ -964,6 +964,11 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 				this.getDatasetMeta(element._datasetIndex).controller[method](element);
 			}
 		}
+
+		if (mode === 'dataset') {
+			method = enabled ? 'setDatasetHoverStyle' : 'removeDatasetHoverStyle';
```

Comment:
It seems like `setDatasetHoverStyle` should be defined on most types and not just `line`, so that this doesn't fail with other types

Reply:
Its defined as `setDatasetHoverStyle: helpers.noop` in `DatasetController` that all datasets should be based on. So it should not fail.

Reply:
Oh yeah, I missed that. Still might be nice to add to bar charts, etc. at some point, but I suppose it's not a blocker